kasa/tests/conftest.py

```diff
@@ -370,6 +377,12 @@ def pytest_addoption(parser):
     parser.addoption(
         "--ip", action="store", default=None, help="run against device on given ip"
     )
+    parser.addoption(
+        "--username", action="store", default=None, help="tapo authentication username"
```

Comment:
```suggestion
        "--username", action="store", default=None, help="authentication username"
```
As this is not only tapo-specific, let's change the help text.